test/correct-filename.test.js

```diff
@@ -1,73 +1,70 @@
-var should = require("should");
-var path = require("path");
-var fileLoader = require("../");
+const fileLoader = require('../');
```

Comment:
Maybe we end up with both. Unit tests below `src` and fixtures/integration tests below `test`.